### PR TITLE
Update Retention policies with content from optimum setup book

### PR DESCRIPTION
**Pull request by harrisonmeister**

### CODE REVIEW

File: docs/administration/retention-policies/index.md [hunk 1]
```diff
@@ -2,54 +2,69 @@
...
 
-The Octopus Server settings delete **releases** from the database. This is a data deletion. It also cleans up any **artifacts**, **deployments, tasks, events** and **logs** attached to the release. Releases that are still on the overall dashboard are never deleted. It is assumed to be the working release and may still be promoted (even if their dates fall well outside the retention policy). No packages from the internal package repository will be deleted as part of this policy, but they may be deleted by a corresponding repository retention policy.
+### Tentacle files {#targets-whats-deleted}
 
-### Tentacle files
+The Tentacle settings delete **packages**, and expanded **files and folders** from packages on the Tentacle machine that is being deployed to. Note that if you use the [Custom Installation Directory ](/docs/deployment-process/configuration-features/custom-installation-directory.md) feature, we will never delete from that directory during retention policies as it's assumed this directory has a working release in it. This can be purged during deployment in the project step settings.
```
Comment (octocrock) [inline docs/administration/retention-policies/index.md:34]: How about:  
>The Tentacle settings delete **packages**, and **files and folders** extracted from packages on the Tentacle machine that is being deployed to.

File: docs/administration/retention-policies/index.md [hunk 2]
```diff
@@ -65,17 +80,48 @@ You can keep all, or select a number of releases to keep.
...
 
-## Built-in feed retention policy {#RetentionPolicies-builtinfeed-retentionpolicy}
+## Set Built-in feed retention policy {#set-builtinfeed-retentionpolicy}
 
 You can find the built-in repository retention policy settings under **{{Library,Packages}}**.
 
 ![](images/3278060.png "width=500")
 
-This can also be set to keep a set number, or keep for a set number of days:
+This can also be set to keep packages forever, or for a set number of days:
```
Comment (octocrock) [inline docs/administration/retention-policies/index.md:89]: Should there be a caveat here for Cloud?

File: docs/administration/retention-policies/retention-policy-tentacle-cleanup-and-troubleshooting.md [hunk 3]
```diff
@@ -95,13 +93,13 @@ See below the messages you will have in your raw deployment logs at the end of a
 
 ```
 
-## Package and extraction directories {#RetentionpolicyTentaclecleanupandtroubleshooting-Packageandextractiondirectories}
+## Package and extraction directories {#retention-policy-package-directories}
 
-You can find your packages under C:\Octopus\files (or c:\Octopus\[Instance Name])\files)
+You can find your packages under `C:\Octopus\files`, or `C:\Octopus\[Instance Name])\files` if you have multiple Tentacle instances on one machine.
```
Comment (octocrock) [inline docs/administration/retention-policies/retention-policy-tentacle-cleanup-and-troubleshooting.md:98]: What about for Linux tentacles?

**Review by octocrock — CHANGES_REQUESTED** [review]

File: docs/administration/retention-policies/index.md (hunk 2 above)
Comment (harrisonmeister) [reply docs/administration/retention-policies/index.md]: Agreed, hint added (and to the release section too :smile:)

File: docs/administration/retention-policies/retention-policy-tentacle-cleanup-and-troubleshooting.md (hunk 3 above)
Comment (harrisonmeister) [reply docs/administration/retention-policies/retention-policy-tentacle-cleanup-and-troubleshooting.md]: Added linux directories

File: docs/administration/retention-policies/index.md (hunk 1 above)
Comment (harrisonmeister) [reply docs/administration/retention-policies/index.md]: :heart:

File: docs/administration/retention-policies/index.md [hunk 4]
```diff
@@ -2,54 +2,69 @@
...
-As you deploy more often and to different environments, files and releases can build up. This is what retention policies are for. They allow you to define what is kept in terms of releases, packages and files.  
+As you deploy more often and to different environments, files and releases can build up. Octopus handles this using retention policies. They allow you to control how Octopus decides which releases, packages and files are kept.   
+
+## What is deleted {#what-is-deleted}
+
+There are a number of different types of retention policies that run. Those on the Octopus Server, those on the Tentacle and those in the built-in package repository.
+
+### Releases {#releases-whats-deleted}
+
+The Octopus Server settings delete **releases** from the database. This is a data deletion. It also cleans up any **artifacts**, **deployments, tasks, events** and **logs** attached to the release. No packages from the internal package repository will be deleted as part of this policy, but they may be deleted by a corresponding repository retention policy.
```
[the remark this answers is not in the document]
Comment (harrisonmeister) [reply docs/administration/retention-policies/index.md]: :heart:

File: docs/administration/retention-policies/index.md [hunk 5]
```diff
@@ -2,54 +2,69 @@
...
 position: 90
+hideInThisSectionHeader: true
 ---
 
-As you deploy more often and to different environments, files and releases can build up. This is what retention policies are for. They allow you to define what is kept in terms of releases, packages and files.  
+As you deploy more often and to different environments, files and releases can build up. Octopus handles this using retention policies. They allow you to control how Octopus decides which releases, packages and files are kept.   
+
+## What is deleted {#what-is-deleted}
+
+There are a number of different types of retention policies that run. Those on the Octopus Server, those on the Tentacle and those in the built-in package repository.
```
[the remark this answers is not in the document]
Comment (harrisonmeister) [reply docs/administration/retention-policies/index.md]: :heart:

**Review by octocrock — APPROVED** [review]

**Pull request merged**